### DISCUSSION TITLE
Can not move photos into album containing apostrophe in title

### DISCUSSION THREAD

**Discussion by vjuliano** [LycheeOrg/Lychee#2441] [Bugs] [answered]
### Lychee version

5.3.0

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

If you create an album with a name such as "Alex's Photos",   the 'move' album list will not allow you to move photos to that album.  Selecting the album title does nothing.

![Screenshot 2024-05-20 at 8 14 50 PM](https://github.com/LycheeOrg/Lychee/assets/58374187/fbf7bc2b-b97a-45ab-9774-5e77fe8331e0)
![Screenshot 2024-05-20 at 8 20 11 PM](https://github.com/LycheeOrg/Lychee/assets/58374187/1e9a092d-5b4b-48d1-99e6-dbe0fd9c38e8)

### Steps to reproduce the issue

1.  Create album with title containing apostrophe.
2. Select photos elsewhere and perform 'Move'
3. Filter list and attempt to select album title.
4. Unable to perform move, and JS crash stack trace in console.

### Diagnostics [REQUIRED]

    Diagnostics
    -----------
        Warning: Dropbox import not working. dropbox_key is empty.
        Info: Found 17 medium that could be generated.
              You can use `php artisan lychee:generate_thumbs medium 17` to generate them.
        Info: Found 2 medium2x that could be generated.
              You can use `php artisan lychee:generate_thumbs medium2x 2` to generate them.
        Info: Found 6 small images without filesizes.
              You can use `php artisan lychee:variant_filesize 6` to compute them.
    

    System Information
    ------------------
        Lychee Version (git):                    local (6a70970) -- Could not compare.
        DB Version:                              5.3.0
        
        composer install:                        --no-dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 default
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        true
        
        System:                                  Linux
        PHP Version:                             8.3.7
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                America/Los_Angeles
        Max uploaded file size:                  200M
        Max post size:                           200M
        Livewire chunk size:                     12.00 MB
        Max execution time:                      200
        MySQL Version:                           11.2.3-MariaDB-1:11.2.3+maria~ubu2304
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1691
        GD Version:                              2.3.3
        Number of foreign key:                   79 found.
    

    Config Information
    ------------------
        version:                                 050300
        check_for_updates:                       1
        sorting_photos_col:                      created_at
        sorting_photos_order:                    DESC
        sorting_albums_col:                      max_taken_at
        sorting_albums_order:                    DESC
        imagick:                                 1
        skip_duplicates:                         1
        small_max_width:                         0
        small_max_height:                        360
        medium_max_width:                        1920
        medium_max_height:                       1080
        lang:                                    en
        image_overlay_type:                      none
        default_license:                         CC-BY-NC-ND-4.0
        compression_quality:                     90
        grants_full_photo_access:                1
        delete_imported:                         1
        mod_frame_enabled:                       1
        mod_frame_refresh:                       30
        thumb_2x:                                1
        small_2x:                                1
        medium_2x:                               1
        landing_page_enable:                     1
        force_migration_in_production:           1
        search_public:                           0
        hide_version_number:                     0
        SL_enable:                               0
        SL_for_admin:                            0
        recent_age:                              60
        grants_download:                         1
        photos_wraparound:                       0
        map_display:                             1
        zip64:                                   1
        map_display_public:                      1
        map_provider:                            Wikimedia
        force_32bit_ids:                         0
        map_include_subalbums:                   1
        update_check_every_days:                 3
        has_exiftool:                            1
        share_button_visible:                    1
        import_via_symlink:                      0
        has_ffmpeg:                              1
        location_decoding:                       1
        location_decoding_timeout:               30
        location_show:                           1
        location_show_public:                    1
        rss_enable:                              0
        rss_recent_days:                         7
        rss_max_items:                           100
        prefer_available_xmp_metadata:           1
        editor_enabled:                          1
        lossless_optimization:                   1
        swipe_tolerance_x:                       150
        swipe_tolerance_y:                       250
        local_takestamp_video_formats:           .avi|.mov
        log_max_num_line:                        10000
        unlock_password_photos_with_url_param:   0
        nsfw_visible:                            0
        nsfw_blur:                               1
        nsfw_warning:                            1
        nsfw_warning_admin:                      1
        nsfw_banner_override:                    
        map_display_direction:                   1
        album_subtitle_type:                     oldstyle
        upload_processing_limit:                 4
        new_photos_notification:                 1
        legacy_id_redirection:                   1
        zip_deflate_level:                       6
        SA_enabled:                              1
        default_album_protection:                1
        allow_username_change:                   1
        album_decoration:                        layers
        album_decoration_orientation:            row
        auto_fix_orientation:                    1
        use_job_queues:                          0
        random_album_id:                         starred
        use_last_modified_date_when_no_exif_date: 0
        layout:                                  justified
        date_format_photo_thumb:                 M j, Y, g:i:s A e
        date_format_photo_overlay:               M j, Y, g:i:s A e
        date_format_sidebar_uploaded:            M j, Y, g:i:s A e
        date_format_sidebar_taken_at:            M j, Y, g:i:s A e
        date_format_hero_min_max:                F Y
        date_format_hero_created_at:             M j, Y, g:i:s A T
        date_format_album_thumb:                 M Y
        upload_chunk_size:                       0
        nsfw_banner_blur_backdrop:               0
        search_pagination_limit:                 1000
        search_minimum_length_required:          4
        photo_layout_justified_row_height:       320
        photo_layout_masonry_column_width:       300
        photo_layout_grid_column_width:          250
        photo_layout_square_column_width:        200
        photo_layout_gap:                        12
        display_thumb_album_overlay:             always
        display_thumb_photo_overlay:             hover
        use_album_compact_header:                0
        default_album_thumb_aspect_ratio:        1/1
        login_button_position:                   left
        back_button_enabled:                     0
        back_button_text:                        Return to Home
        current_job_processing_visible:          1
        SA_random_thumbs:                        0

### Browser & System [REQUIRED]

Firefox 126.0 
Mac OS 14.5 and Windows 11

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Good spot!
I will see for a quick patch.
